### PR TITLE
[Clustering] Small tidy up - removed unusued variables

### PR DESCRIPTION
**Pull request by wwwind**
Small tidy up: removed variables that are not set and not used - some leftovers.

### CODE REVIEW

**Review by akarmi — APPROVED** [review]

File: tensorflow_model_optimization/python/core/clustering/keras/cluster_wrapper.py [hunk 1]
```diff
@@ -326,11 +326,11 @@ def trainable(self, value):
 
   @property
   def trainable_weights(self):
-    return self.layer.trainable_weights + self._trainable_weights
```
Comment (alanchiao) [inline tensorflow_model_optimization/python/core/clustering/keras/cluster_wrapper.py:329]: Maybe I've misunderstood but self._trainable_weights and self._non_trainable_weights should be updated automatically during calls to self.add_weight(). 

By removing "self._trainable_weights", someone calling layer.trainable_weights will no longer see all the weights that your wrapper is adding, which breaks the interface. 

You can try creating a unit test that checks what layer.trainable_weights outputs and see if it makes sense and look at the base tf.keras.layers.Layer implementation to see what it does.

**Comment by alanchiao** [comment]
Marking as not ready to pull given my above comment. Please do add back if it changes.

**Comment by alanchiao** [comment]
This was accidentally merged - please followup on the above.

**Comment by wwwind** [comment]
Hi @alanchiao I will investigate. During debugging I have not seen them updating, but I will re-test this.

**Comment by benkli01** [comment]
Hi @alanchiao and @wwwind , 
as I have encountered some problems with this, I will raise a PR to revert this commit.

**Pull request merged**